packages/css/src/components/anchor/anchor.template.ts

```diff
@@ -0,0 +1,54 @@
+import { Anchor } from '@dso-toolkit/sources';
+import { html, nothing } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined';
+
+import { iconTemplate } from '../icon/icon.template';
+
+export function anchorTemplate({ icon, iconAfter, iconOnly, label, modifier, url }: Anchor) {
+  return html`
+    <a href=${url}
+       title=${ifDefined(iconOnly ? label : undefined)}
+       class=${ifDefined(modifier)}
+       target=${ifDefined(modifier && modifier.indexOf('extern') > -1 ? '_blank' : undefined)}
```

Comment:
TypeScript geeft ons het volgende (betere) alternatief:

```suggestion
       target=${ifDefined(modifier?.indexOf('extern') > -1 ? '_blank' ?? undefined)}
```

* `?.` https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining
* `??` https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
Het voorstel met `??` gaat in dit geval niet werken, aangezien we ook een ternary starten verwacht hij ergens een `:`

Reply:
Je hebt gelijk!